### DISCUSSION TITLE
Allow empty tabs to act as roots of the tree

### DISCUSSION THREAD

**Discussion by noneuclideanmotion** [mbnuqw/sidebery#421] [Ideas]
For example, I have a `videos` panel. It would be great if I could separate all programming videos with a pseudo-tab that I could rename as `programming* and then all my programming videos could be a child of the programming pseudo-tab (perhaps it could just be a new tab which you can rename)

**Comment by Kelendral-gh** [comment]
Select tab > Right-Click > Group
